src/app/design/guidelines/error-handling/index.html

```diff
@@ -103,5 +103,22 @@
 			</div>
 		</stache-column>
 	</stache-row>
+	
+	<stache-page-anchor>
+		Background process errors
+	</stache-page-anchor>
+
+	<stache-row>
+		<stache-column screenSmall="6">
+			<p>
+				Since background processes run outside the current page context, use a <a routerLink="/components/toast">toast</a> to notify the user of an error with a background process. The toast will show up regardless of which page of the system the user is on and persist as they navigate through the system.
```

Comment:
I'd change this to use plurals to avoid switching from plural to singular and for consistency with the other items on the page: "Since background processes run outside the current page context, use toasts to notify users of errors with background processes. Toasts appear regardless of which page users are on and persist as they navigate through the system."